teos/api.py

```diff
@@ -107,6 +109,8 @@ def __init__(self, inspector, internal_api_endpoint):
         self.app = Flask(__name__)
         self.inspector = inspector
         self.internal_api_endpoint = internal_api_endpoint
+        self.channel = grpc.insecure_channel(internal_api_endpoint)
```

Comment:
Do we need to store the channel?

Reply:
Removed (also in rpc and teos_cli) in 5988290.